doc/reference.md

```diff
@@ -567,6 +569,21 @@ some-data.data : Array3 some-data.len S32
 //                                      in type of the projected term
 ```
 
+## Booleans
+
+The boolean type is: `Bool` with values `true` and `false`.
```

Comment:
I must admit I never really know when to use the term ‘value’ now, but I think this is ok given that these terms evaluate to these values? Might be something to ask type theory twitter about heh 🤔